docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -86,6 +86,23 @@ The following features are **not** certified or supported in this release:
 
 For up-to-date information about the features of WebLogic Server that are supported in Kubernetes environments, see My Oracle Support Doc ID 2349228.1.
 
+### Meet Kubernetes resource name restrictions
+          
+Kubernetes requires the names of some resource types to follow the DNS label standard as defined in [DNS Label Names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) and [RFC 1123](https://tools.ietf.org/html/rfc1123). This requirement restricts the characters that are allowed in the names of these resources, and also limits the length of these names to no more than 63 characters.
+
+The following is a list of such Kubernetes resources that the operator generates when a domain resource is deployed, including how their names are constructed.
+          
+* A domain introspector job named as `<domainUID>-<introspectorJobNameSuffix>`. The default suffix is `-introspector`, which can be overridden using the operator's Helm configuration `introspectorJobNameSuffix` (see [WebLogic domain Management]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#weblogic-domain-management" >}})).
+* A ClusterIP type service and a pod for each WebLogic server named as `<domainUID>-<serverName>`. 
+* A ClusterIP type service for each WebLogic cluster named as `<domainUID>-cluster-<clusterName>`.
+* A optional NodePort type service, also known as an external service, for the WebLogic admin server named as `<domainUID>-<adminServerName>-<externalServiceNameSuffix>`. The default suffix is `-ext`, which can be overridden using the operator's Helm configuration `externalServiceNameSuffix` (see [WebLogic domain Management]({{< relref "/userguide/managing-operators/using-the-operator/using-helm#weblogic-domain-management" >}})).
+
+The operator puts in place certain validation checks and conversion to prevent these resources from violating Kubernetes restrictions.
+* All the names that are discussed above can only contain the characters `A-Z`, `a-z`, `0-9`, `-`, or `_`, and must start and end with with an alphanumeric character. Note that when generating pod and service names, the operator will convert configured names to lower case and substitute a hyphen (`-`) for each underscore (`_`).
```

Comment:
All the names that are discussed above -> All the names previously described (Style guidelines dictate, "Do not use above to refer to material that can be found at an earlier position in a document. Formatting changes can cause this instruction to mislead the reader. Use preceding, previous, or earlier instead."